brainfuck/commands.go

```diff
@@ -0,0 +1,68 @@
+package brainfuck
+
+import "fmt"
+
+/*
+	Move buffer's pointer one step forward ('>' operator).
```

Comment:
Doc comment should start with the member name it describes.
Fix in other places too.

```suggestion
	forward  ..... Move buffer's pointer one step forward ('>' operator).
```

Reply:
fixed